Dockerfile.minimal

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.7-alpine AS builder 
+
+ENV DISTRIBUTION_DIR /go/src/github.com/docker/distribution
+ENV DOCKER_BUILDTAGS include_oss include_gcs
+
+RUN set -ex \
+    && apk add --no-cache make git gcc libc-dev
+
+WORKDIR $DISTRIBUTION_DIR
+COPY . $DISTRIBUTION_DIR
+
+RUN make PREFIX=/go clean binaries
+
+
+FROM alpine:3.5
+
+RUN set -ex \
+    && apk add --no-cache ca-certificates apache2-utils
+
+ENV PATH /go/bin:$PATH
+COPY --from=builder /go/bin/registry /go/bin/registry
```

Comment:
why modify `PATH`? Instead of copying to `/go/bin`, this could just copy to a standard party (`/bin/`, `/usr/local/bin` - whichever makes more sense)